### PR TITLE
[Doc Improvement][Build API-based message extension][3860161]

### PR DESCRIPTION
**Pull request by v-bvishnu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-api-msg-ext-ttk.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/47219705e457bc2bc4b9216d2220d5ce843b9d6a/msteams-platform%2Fsbs-api-msg-ext-ttk.yml) | 82 | 72 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2a0e3565-cff0-4020-8e31-81c230bd2a09) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/create-api-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/879b9c3485f33811939755999c8df08406a93b60/msteams-platform%2Fmessaging-extensions%2Fcreate-api-message-extension.md) | 96 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0c41835c-2974-4526-a062-eb9b7cbb753e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-api-msg-ext-ttk.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/879b9c3485f33811939755999c8df08406a93b60/msteams-platform%2Fsbs-api-msg-ext-ttk.yml) | 82 | 72 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/af99d6d9-b45a-4464-8a0e-8792d1001b15) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/create-api-message-extension.md [hunk 1]
```diff
@@ -113,7 +113,9 @@ To test your API-based message extension created in the Developer Portal for Tea
 # [Visual Studio Code](#tab/visual-studio-code)
 
 > [!NOTE]
-> Teams Toolkit support for API-based message extension is available only in Teams Toolkit pre-release version. Before you get started, ensure that you've installed a [Teams Toolkit pre-release version](../toolkit/install-Teams-Toolkit.md#install-a-pre-release-version).
+> * Teams Toolkit support for API-based message extension is available only in Teams Toolkit pre-release version. Before you get started, ensure that you've installed a [Teams Toolkit pre-release version](../toolkit/install-Teams-Toolkit.md#install-a-pre-release-version).
+>
+> * Teams Toolkit only supports OpenAPI Specification version 3.0.1.
```
Comment (MuyangAmigo) [inline msteams-platform/messaging-extensions/create-api-message-extension.md:119]: Is this accurate? @SLdragon

Comment (SLdragon) [reply msteams-platform/messaging-extensions/create-api-message-extension.md:119]: Teams Toolkit support OpenAPI spec version 2.0, 3.0.x, and 2.0 will be automatically converted to 3.0.x

File: msteams-platform/messaging-extensions/create-api-message-extension.md [hunk 2]
```diff
@@ -113,7 +113,9 @@ To test your API-based message extension created in the Developer Portal for Tea
 # [Visual Studio Code](#tab/visual-studio-code)
 
 > [!NOTE]
-> Teams Toolkit support for API-based message extension is available only in Teams Toolkit pre-release version. Before you get started, ensure that you've installed a [Teams Toolkit pre-release version](../toolkit/install-Teams-Toolkit.md#install-a-pre-release-version).
```
Comment (SLdragon) [inline msteams-platform/messaging-extensions/create-api-message-extension.md:116]: Stable version is already support API-based message extension, and don't need pre-release version any more

File: msteams-platform/sbs-api-msg-ext-ttk.yml [hunk 3]
```diff
@@ -17,7 +17,8 @@ items:
 
     > [!NOTE]
     >
-    > API-based message extensions only support search commands.
+    > * API-based message extensions only support search commands.
+    > * Teams Toolkit only supports OpenAPI Specification version 3.0.1.
```
Comment (SLdragon) [inline msteams-platform/sbs-api-msg-ext-ttk.yml:21]: Same with above

File: msteams-platform/sbs-api-msg-ext-ttk.yml [hunk 4]
```diff
@@ -302,7 +303,11 @@ items:
 
     ```
     </details>
-    
+
+    > [!NOTE]
+    >
+    > Teams Toolkit only supports OpenAPI Specification version 3.0.1.
```
Comment (SLdragon) [inline msteams-platform/sbs-api-msg-ext-ttk.yml:309]: Same with above

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/create-api-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6e6c0bce69c4c44c6a765a2384985341c6898264/msteams-platform%2Fmessaging-extensions%2Fcreate-api-message-extension.md) | 96 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cc4a83d9-1917-4988-bca8-5376fa24ffb8) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-api-msg-ext-ttk.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/6e6c0bce69c4c44c6a765a2384985341c6898264/msteams-platform%2Fsbs-api-msg-ext-ttk.yml) | 81 | 73 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6aec4f9d-1353-49ef-a565-35a555312c50) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/create-api-message-extension.md (hunk 1 above)
Comment (v-bvishnu) [reply msteams-platform/messaging-extensions/create-api-message-extension.md:119]: Updated.

File: msteams-platform/messaging-extensions/create-api-message-extension.md (hunk 2 above)
Comment (v-bvishnu) [reply msteams-platform/messaging-extensions/create-api-message-extension.md:116]: Updated.

File: msteams-platform/sbs-api-msg-ext-ttk.yml (hunk 3 above)
Comment (v-bvishnu) [reply msteams-platform/sbs-api-msg-ext-ttk.yml:21]: Updated.

File: msteams-platform/sbs-api-msg-ext-ttk.yml (hunk 4 above)
Comment (v-bvishnu) [reply msteams-platform/sbs-api-msg-ext-ttk.yml:309]: Updated.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/create-api-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f8ede419b48aca5ae5bc0af8f448893097dbd42b/msteams-platform%2Fmessaging-extensions%2Fcreate-api-message-extension.md) | 95 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/79747b8e-695d-4a52-95c1-c7625f14399a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-api-msg-ext-ttk.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/f8ede419b48aca5ae5bc0af8f448893097dbd42b/msteams-platform%2Fsbs-api-msg-ext-ttk.yml) | 81 | 73 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/84743ab2-8697-4b1d-8a73-48c56327160d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/create-api-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/defd354f8e16008004aa7b3686954028190eb9c0/msteams-platform%2Fmessaging-extensions%2Fcreate-api-message-extension.md) | 96 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e9490c7d-38bf-427d-8a33-0058d1db80a1) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-api-msg-ext-ttk.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/defd354f8e16008004aa7b3686954028190eb9c0/msteams-platform%2Fsbs-api-msg-ext-ttk.yml) | 81 | 73 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/40bbea02-a544-48e5-aef7-44e457bf19cd) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by MuyangAmigo — APPROVED** [review]

**Review by SLdragon — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/create-api-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b7ca2133035151cf466d5010e1750e28a37c527a/msteams-platform%2Fmessaging-extensions%2Fcreate-api-message-extension.md) | 95 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e0eeca68-8d87-4bda-82a7-44a710a5be59) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-api-msg-ext-ttk.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/b7ca2133035151cf466d5010e1750e28a37c527a/msteams-platform%2Fsbs-api-msg-ext-ttk.yml) | 80 | 76 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3f279a29-d831-4984-8bad-79d036fb521e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**